### PR TITLE
[REG-1049] Document custom replay models

### PR DESCRIPTION
**Pull request by RG-nAmKcAz**
See also (https://github.com/Regression-Games/RGUnityBots/pull/18)

### CODE REVIEW

File: docs/studios/unity/unity-sdk/in-editor-replay.mdx [hunk 1]
```diff
@@ -65,4 +65,22 @@ Shows any command(s) that the Bot sent to the game for the current tick.  These
 Shows any validation(s) that the Bot performed on the current tick.  These are normally test cases coded into the Bot to validate that the tick data changed as expected based on prior actions or other factors in the game state.  See [Validation Bots](./creating-bots/validation-bots) for more information.
 
 ### Registering Custom Models For Replay Objects
-... Coming Soon ...
+When viewing a replay, entities that are dynamically spawned into the scene (not defined in the scene itself) need to be told what model to render with.  By default they will load as a pink capsule, but they can be configured with a specific model based on their type.
```
Comment (vontell) [inline docs/studios/unity/unity-sdk/in-editor-replay.mdx:68]:
```suggestion
When viewing a replay, entities that are dynamically spawned into the scene (not defined in the scene itself) need to be told what model to render with. By default they will load as a pink capsule, but they can be configured with a specific model based on their type.
```

File: docs/studios/unity/unity-sdk/in-editor-replay.mdx [hunk 2]
```diff
@@ -65,4 +65,22 @@ Shows any command(s) that the Bot sent to the game for the current tick.  These
...
+:::tip
+The associations from entity type to replay model are saved in your project as `Assets/RegressionGames/Editor/CustomReplayModels.asset`.
+
+You can edit this asset in the inspector directly or by using the `Configure Custom Replay Models` button in the RG Bot Replay window to open the inspector.
+:::
+
+![replay model manager](replay_model_manager.png)
+
+:::tip
+While the associations support any game object, it is best to associate just the model for your entity.
```
Comment (vontell) [inline docs/studios/unity/unity-sdk/in-editor-replay.mdx:83]:
```suggestion
While the associations support any GameObject, it is best to associate just the model for your entity.
```
Not sure if this is more correct?

**Review by vontell — APPROVED** [review]
Approving but just have a few styling suggestions.

File: docs/studios/unity/unity-sdk/in-editor-replay.mdx (hunk 1 above)
Comment (RG-nAmKcAz) [reply docs/studios/unity/unity-sdk/in-editor-replay.mdx:68]: This entire doc uses 2 spaces.  As it should be...  But I know that's a generational thing (get off my lawn :P)

I suppose I can live with 1 space after a period, but I'll need to update everything in this doc to match that style.

**Review by abeizer — APPROVED** [review]
Approving, pending Aaron's comment

**Pull request merged**